templates/enmasse-controller-manager/020-Role-enmasse-controller-manager.yaml

```diff
@@ -5,9 +5,15 @@ metadata:
     app: enmasse
   name: enmasse.io:enmasse-controller-manager
 rules:
+  - apiGroups: [ "" ]
+    resources: [ "configmaps", "secrets", "pods" ]
+    verbs: [ "get", "list", "watch", "update", "create", "patch" ]
+  - apiGroups: [ "enmasse.io" ]
+    resources: [ "addressspaces", "addresses" ]
+    verbs: [ "get", "list", "watch", "update", "create", "patch" ]
   - apiGroups: [ "admin.enmasse.io" ]
     resources: [ "authenticationservice" ]
     verbs: [ "get", "list", "watch", "update", "create", "patch" ]
   - apiGroups: [ "iot.enmasse.io" ]
-    resources: [ "iotconfig" ]
+    resources: [ "iotconfigs", "iotprojects" ]
     verbs: [ "get", "list", "watch", "update", "create", "patch" ]
```

Comment:
I don't see any point in providing namespace access to 'enmasse.io' and 'iot.enmasse.io' resources, as they are already granted by the cluster role.

Reply:
True.